docs/source/rbac/tech-implementation.md

```diff
@@ -43,7 +43,7 @@ Prior to 3.0, tokens stored _roles_,
 which meant their scopes were resolved on each request.
 :::
 
-API tokens grant access to JupyterHub's APIs. The Role Based Access Control (RBAC) framework allows for the requesting of tokens with specific permissions.
+API tokens grant access to JupyterHub's APIs. The [RBAC framework](https://github.com/jupyterhub/jupyterhub/blob/main/docs/source/rbac/index.md) allows for requesting tokens with specific permissions.
```

Comment:
As I suggested before, this should be a local reference (i.e. a relative path, or a myst reference target). I suggested a value that would work: ./index.md

Reply:
> As I suggested before, this should be a local reference (i.e. a relative path, or a myst reference target). I suggested a value that would work: ./index.md

![Capture12](https://user-images.githubusercontent.com/81184842/197738724-b2d05da1-7ff3-4219-b5ed-58656350c261.JPG)
 sir @minrk is above correct?

Reply:
yes